funcx_sdk/funcx/sdk/web_client.py

```diff
@@ -39,7 +39,6 @@ def __init__(
         description: t.Optional[str] = None,
         public: bool = False,
         group: t.Optional[str] = None,
-        searchable: bool = True,
         serializer: t.Optional[FuncXSerializer] = None,
```

Comment:
As this is nominally an interface, suggest accepting the argument as is, but ignoring it&nbsp;&mdash;&nbsp;as you've done&nbsp;&mdash;&nbsp;below.  Can optionally `warnings.warn` as well, and we can plan to fully remove it from the signature in give or take 6 months.